### ISSUE TITLE
AG WG Review: WCAG2ICT update to align with EN 301 549

### ISSUE THREAD

**Issue by maryjom** [w3c/wcag2ict#750]
## Summary
The WCAG2ICT Task Force has made updates to align guidance with EN 301 549 currently being drafted for the European Accessibility Act. This update includes a number of editorial changes as well as some new and changed guidance listed in the [Content for Review](#Content-for-Review) section below. 

We are asking for an AG WG review to be followed by a CfC to republish the document. This will give EN 301 549 a consistent, harmonized WCAG2ICT Note to reference in their standard.

## Maturity Level
Ready to publish

## Action Needed

1. Review content (See Content for Review section below for details.)
2. Use the thumbs up or thumbs down icons on this description comment to indicate agreement or disagreement with the changes.  Comments aren't needed unless you have edits or concerns.
3. If you have edits or concerns, add a comment to this issue to provide suggested text and reasoning/justification for the change. We'd like to keep editing to a minimum, if possible, to minimize further impact on EN 301 549.

----

## Background information 

The last time the WCAG2ICT Note was published, the WCAG2ICT Task Force had limitations imposed on the guidance that it was permitted to contain. The proposed updates to the WCAG2ICT Note utilizes the work statement expansion in the scope and, for the first time, suggests more substantive changes to the WCAG language for 1.4.4 Resize Text when applied in a non-web technology context. The vast majority of changes are purely editorial. Other changes in the note are to make the verbiage more consistent as well as harmonize some of the explanatory notes where there was different guidance in EN 301 549 and WCAG2ICT, as well as reduce their effort to split the guidance between non-web documents and non-web software. 

Once this update is reviewed by the working Group, there will be a Call for Consensus to publish this as an updated WCAG2ICT Note.

## Content for Review

* [Diff view](https://services.w3.org/htmldiff?doc1=https%3A%2F%2Fwww.w3.org%2FTR%2Fwcag2ict-22%2F&doc2=https%3A%2F%2Fw3c.github.io%2Fwcag2ict%2F) showing changes between the officially published WCAG2ICT Group Note and the Editor's draft that the TF would like to publish.
* [Editor's draft](https://w3c.github.io/wcag2ict/) (no markup of changes)
* List of substantive changes with a link to the PR can be found in the section titled [Changes since the 15 November 2024 Note](https://w3c.github.io/wcag2ict/#changelog) are summarized below with links to the that changed section in the diff file.
    * [Key term "content"](https://services.w3.org/htmldiff?doc1=https%3A%2F%2Fwww.w3.org%2FTR%2Fwcag2ict-22%2F&doc2=https%3A%2F%2Fw3c.github.io%2Fwcag2ict%2F#content-on-and-off-the-web): Added Notes 2 and 3     
    * [1.3.1 Info and Relationships](https://services.w3.org/htmldiff?doc1=https%3A%2F%2Fwww.w3.org%2FTR%2Fwcag2ict-22%2F&doc2=https%3A%2F%2Fw3c.github.io%2Fwcag2ict%2F#applying-sc-1-3-1-info-and-relationships-to-non-web-documents-and-software): Added Note 1 from the EN 301 549.  
    * 1.4.2, 2.1.2, 2.2.2, and 2.3.1: Adjust note to use consistent verbiage and avoid use of "must". All the SCs now have a note that is exactly the same as Note 1 of [1.4.2 Audio Control](https://services.w3.org/htmldiff?doc1=https%3A%2F%2Fwww.w3.org%2FTR%2Fwcag2ict-22%2F&doc2=https%3A%2F%2Fw3c.github.io%2Fwcag2ict%2F#applying-sc-1-4-2-audio-control-to-non-web-documents-and-software) and the replacement language was adjusted to indicate the changes to this note.
    * [1.4.4 Resize text](https://services.w3.org/htmldiff?doc1=https%3A%2F%2Fwww.w3.org%2FTR%2Fwcag2ict-22%2F&doc2=https%3A%2F%2Fw3c.github.io%2Fwcag2ict%2F#applying-sc-1-4-4-resize-text-to-non-web-documents-and-software): Adjusted language where platforms provide some, but not all text that scales by 200%, added Note 1 (which was in the 2013 WCAG2ICT)  and Notes 2 and 3 were edited for clarity.
    * [1.4.10 Reflow](https://services.w3.org/htmldiff?doc1=https%3A%2F%2Fwww.w3.org%2FTR%2Fwcag2ict-22%2F&doc2=https%3A%2F%2Fw3c.github.io%2Fwcag2ict%2F#applying-sc-1-4-10-reflow-to-non-web-documents-and-software): Added Note 5 for non-web documents (text originated from note 8) 
    * [2.1.1 Keyboard](https://services.w3.org/htmldiff?doc1=https%3A%2F%2Fwww.w3.org%2FTR%2Fwcag2ict-22%2F&doc2=https%3A%2F%2Fw3c.github.io%2Fwcag2ict%2F#applying-sc-2-1-1-keyboard-to-non-web-documents-and-software): Added a pre-condition, edited Notes 1, 4, and 5 for clarity and to align more closely with EN 301 549 notes, and added Notes 2 and 3.
    * [2.2.2 ](https://services.w3.org/htmldiff?doc1=https%3A%2F%2Fwww.w3.org%2FTR%2Fwcag2ict-22%2F&doc2=https%3A%2F%2Fw3c.github.io%2Fwcag2ict%2F#applying-sc-2-2-2-pause-stop-hide-to-non-web-documents-and-software): Updated Note 5 to give full understanding doc name and link and not use "informative" which has a specific meaning in WCAG
    * [3.1.2 Language of Parts](https://services.w3.org/htmldiff?doc1=https%3A%2F%2Fwww.w3.org%2FTR%2Fwcag2ict-22%2F&doc2=https%3A%2F%2Fw3c.github.io%2Fwcag2ict%2F#applying-sc-3-1-2-language-of-parts-to-non-web-documents-and-software): - Add modified EN 301 549 note as Note 2
    * [4.1.2 ](https://services.w3.org/htmldiff?doc1=https%3A%2F%2Fwww.w3.org%2FTR%2Fwcag2ict-22%2F&doc2=https%3A%2F%2Fw3c.github.io%2Fwcag2ict%2F#applying-sc-4-1-2-name-role-value-to-non-web-documents-and-software): Include a word replacement for "user agent" in the success criterion language
    * [Definition of "target"](https://services.w3.org/htmldiff?doc1=https%3A%2F%2Fwww.w3.org%2FTR%2Fwcag2ict-22%2F&doc2=https%3A%2F%2Fw3c.github.io%2Fwcag2ict%2F#applying-target-to-non-web-documents-and-software) - Simplified word replacement in "target" definition to use "content" as the word replacement for "page"

* Notable editorial changes:
    * Incorporate changes from WCAG 2.2 December 2024
    * Consistent use of "non-web", "ICT with closed functionality",  "satisfy"
    * Clearer indication of notes that apply only to non-web documents or to non-web software
    * Improved visual indication of links in word replacement phrases by making only the link text bold
    * Some notes were split up and some were simply moved to keep non-web document-only notes together and non-web software-only notes together. One example is 1.4.12 Text Spacing where Note 2 was designated as software-only and moved to the bottom, and Note 3 was split into Notes 2 and 3 - one note for non-web documents and one for non-web software.

## Reference Material
* Previously published [WCAG2ICT Note November 2024](https://www.w3.org/TR/wcag2ict-22/)

**Comment by maryjom** [comment]
The list of issues in the previous comment have all been merged into the editor's draft. Then @daniel-montalvo will copy into his 2025-Note branch for publication.

**Comment by mgifford** [comment]
Nice changes @maryjom one thing that occurred to me is that we don't mention which version of the EN 301 549. 

Since it is an evolving standard, like most things in the W3C, it might be useful to at least somewhere reference what version is being referred to. Much like the document doesn't talk about Section 508 but the Revised Section 508. 

Doesn't need to be done more than once, and when a new version of EN 301 549 is released (which should be soon) it is good to verify that the advice is still aligned, and update that number.

**Comment by mraccess77** [comment]
For SC 1.4.10 Note 5 - this doesn't seem to align with the updates to the understanding document for SC 1.4.10 which interpret content blocks at 320 rather than the viewport as a whole. For example, a 3 column document could pass even if a document viewer didn't support 320 CSS pixels because each block of text was less than 320 CSS pixels wide.

**Comment by maryjom** [comment]
@mraccess77 For 1.4.4, it is not the intent that if the platform doesn't support any text scaling to 200% to allow content to pass, it's to allow passing where the platform may provide non-linear text scaling for things that have a large font size. Take a look at the [research from MATF](https://github.com/w3c/matf/issues/3#issuecomment-2490573410) on iOS and Android where the larger the text is to start with, the less it is scaled and will never reach 200%. It seems unreasonable to require every single mobile app to implement its own text scaling so that you can reach 200% for all text.  The adjustment we're suggesting is to address that without calling out specific technologies and the way they scale text.

**Comment by GreggVan** [comment]
@maryjom   I understand the intent.  But I you think about it - this is not within our charge to do this.   We essentially are EITHER\
- putting normative changes into a note\
OR\
- we go beyond  saying whether this applies or not - and deciding to say that a new one should be written that is different than WCAG and here is what it should be.

I THINK (correct me if I  am wrong) 

However - there is something here that we need to do beyond saying either "this applies as written - which we do not believe" or saying "this doesn't" - but not explaining it 

How about something like    

> Some platforms such as some mobile platforms do not provide built-in utilities to allow scaling of all text to 200% whereas all browsers did when the SC was written.  The task force felt that the intent was to support scaling up to at least 200% by the platform, not to require that all software provide their own scaling function.  However, if there is no scaling function in the platform at all then text scaling should be provided to ensure that text is large enough for those with 20/40 vision (the equiv of 200% of 20/20.  Note that this SC is also listed for special consideration in closed products.
